installation_guidelines.md

```diff
@@ -71,10 +71,24 @@
     ![phpmyadmin_3](https://user-images.githubusercontent.com/16044116/135674921-e9992ded-6897-49df-8d62-d334fce2a214.png)
     - 
     ![phpmyadmin_4](https://user-images.githubusercontent.com/16044116/135674927-a5ca269a-3e76-4831-8e9b-0d218e050270.png)
+    
+- Add `.env` file
```

Comment:
No need to add `.env` file, since we have the required credentials inside `.env_sample` , renaming `.env_sample` to `.env` would be enough